pyproject.toml

```diff
@@ -1,20 +1,21 @@
 [build-system]
-requires = ["setuptools>=64"]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling", "uv-dynamic-versioning"]
+build-backend = "hatchling.build"
+
+[tool.uv-dynamic-versioning]
+pattern = "default-unprefixed"
 
 [project]
-name = "dicomweb-client"
+name = "pydicomweb-client"
```

Comment:
Ah, first, I thought it was not that good to call the repo "pydicomweb" and the package "pydicomweb-client." But after a second thought, it could be a good idea. Maybe we can come up with a "pydicomweb-server" someday.